lite/kernels/CMakeLists.txt

```diff
@@ -15,3 +15,4 @@ add_subdirectory(apu)
 add_subdirectory(bm)
 add_subdirectory(rknpu)
 add_subdirectory(huawei_ascend_npu)
+add_subdirectory(imagination_nna)
```

Comment:
remove the binary file below

No binary file is allowed to merge to the codebase.